example-utils/usubscription-static-file/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "usubscription-static-file"
+rust-version.workspace = true
+version.workspace = true
+repository.workspace = true
+homepage.workspace = true
+edition.workspace = true
+keywords.workspace = true
+license.workspace = true
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+async-std = { workspace = true, features = ["unstable"] }
```

Comment:
Not an issue, but fyi: up-rust, zenoh and the (upcoming) usubscription-rust are all using tokio in the meantime - mixing runtimes should (and usually does) work, so really just an fyi at this point.

Reply:
Yeah, we should be able to mix and match, but for consistency's sake I opened up #11 a while ago to make them consistent.